### PR TITLE
improvement(db-events): reduce severity to warning for cql_server con…

### PR DESCRIPTION
**Pull request by bentsi**
…nection processing error

Reduce severity to warning for the error message:
'cql_server - exception while processing connection: std::system_error'

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [ ] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

File: sdcm/cluster.py [hunk 1]
```diff
@@ -371,25 +371,25 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
...
+            DatabaseLogEvent(type='FILESYSTEM_ERROR', regex='filesystem_error'),
+            DatabaseLogEvent(type='STACKTRACE', regex='stacktrace'),
+            DatabaseLogEvent(type='BACKTRACE', regex='backtrace', severity=Severity.ERROR),
+            DatabaseLogEvent(type='SEGMENTATION', regex='segmentation'),
+            DatabaseLogEvent(type='INTEGRITY_CHECK', regex='integrity check failed'),
+            DatabaseLogEvent(type='REACTOR_STALLED', regex='Reactor stalled'),
+            DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out'),
+            DatabaseLogEvent(type='BOOT', regex='Starting Scylla Server', severity=Severity.NORMAL),
+            DatabaseLogEvent(type='SUPPRESSED_MESSAGES', regex='journal: Suppressed', severity=Severity.WARNING),
+            DatabaseLogEvent(type='CQL_SERVER_CONN_SYSTEM_ERROR', severity=Severity.WARNING,
```
Comment (fruch) [inline sdcm/cluster.py]: You should put this new one at the top of the list, otherwise it be raised as DATABASE_ERROR

**Comment by roydahan** [comment]
In general, I think this should be taken out of cluster.py and managed in a yaml/JSON/other format file.
We should put some thinking on collisions, what wins, ordering or highest severity, for example.

**Comment by bentsi** [comment]
@roydahan let's create a task for it

File: sdcm/cluster.py [hunk 2]
```diff
@@ -369,26 +369,25 @@ def __init__(self, name, parent_cluster, ssh_login_info=None, base_logdir=None,
...
-                                                        severity=Severity.WARNING),
-                                       DatabaseLogEvent(type='SEMAPHORE_TIME_OUT', regex='semaphore_timed_out'),
-                                       DatabaseLogEvent(type='BOOT', regex='Starting Scylla Server',
-                                                        severity=Severity.NORMAL),
-                                       DatabaseLogEvent(type='SUPPRESSED_MESSAGES', regex='journal: Suppressed',
-                                                        severity=Severity.WARNING)]
+        self._database_error_events = [
+            DatabaseLogEvent(type='NO_SPACE_ERROR', regex='No space left on device'),
+            DatabaseLogEvent(type='UNKNOWN_VERB', regex='unknown verb exception', severity=Severity.WARNING),
+            DatabaseLogEvent(type='CQL_SERVER_CONN_SYSTEM_ERROR', severity=Severity.WARNING,
```
Comment (fruch) [inline sdcm/cluster.py]: should that be more specific ? i.e. it was `broken pipe` or something like that ?

Comment (bentsi) [reply sdcm/cluster.py]: let's catch all of these connection `system_error`-s

**Pull request merged**